client/my-sites/checkout/checkout/redirect-payment-box.jsx

```diff
@@ -284,6 +337,23 @@ export class RedirectPaymentBox extends PureComponent {
 		);
 		const showPaymentChatButton = this.props.presaleChatAvailable && hasBusinessPlanInCart;
 
+		// Wechat qr codes get set on desktop instead of redirecting
+		if ( this.state.qrCodeUrl ) {
+			this.setSubmitState( {
+				info: translate( 'Please scan the WeChat Payment barcode.', { context: 'Instruction to scan the on screen barcode.' } ),
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?